src/Streamly/Internal/FileSystem/Event/Darwin.hs

```diff
@@ -1020,4 +1020,11 @@ showEvent ev@Event{..} =
         where showev f str = if f ev then "\n" ++ str else ""
 #else
 module Streamly.Internal.FileSystem.Event.Darwin () where
+#warning "Autoconf did not find the definition \
+kFSEventStreamCreateFlagFileEvents in Darwin header files.\
+Do you have Cocoa framework header files installed?\
+Not compiling the Streamly.Internal.FileSystem.Event module. \
```

Comment:
Where is `Streamly.Internal.FileSystem.Event`?

Reply:
Should be `Streamly.Internal.FileSystem.Event.Darwin`.